tests/e2e_tests/test_find.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from playwright.sync_api import Page, expect
+
+from tests.e2e_tests.helpers import login_via_magic_link, lookup_find_download_link_for_user_in_govuk_notify
+from tests.e2e_tests.pages.find import DownloadDataPage, FindRequestDataPage
+
+pytestmark = pytest.mark.e2e
+
+
+def test_find_download(page: Page):
+    email_address = login_via_magic_link(page, "test_find_download", email_domain="communities.gov.uk")
+
+    request_data_page = FindRequestDataPage(page)
+    request_data_page.navigate()
+
+    request_data_page.filter_funds("High Street Fund")
+    request_data_page.filter_regions("North West")
+    request_data_page.filter_organisations("Bolton Council")
+    request_data_page.filter_outcomes("Health & Wellbeing", "Transport")
+
+    request_data_page.select_file_format("XLSX (Microsoft Excel)")
+
+    request_data_page.request_data_button.click()
```

Comment:
Maybe a method `request_data` would be more elegant and consistent with the level of abstraction provided by the methods used above; this also removes need to define custom `__init__` method in `FindRequestDataPage`. For reference:

```
class FindRequestDataPage(BasePage):
   ...
   def request_data(self):
      request_data_button = self.page.get_by_role("button", name="Confirm and request data")
      request_data_button.click()
   ...
```